### DISCUSSION TITLE
[A-L] (2023/24) Foglio 16 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#292] [Esercizi - Viaggi/Piazza]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/73c95315-50d6-4b40-bca7-90f292fb1cd3)

**Comment by Elia-Belli** [comment]
Eseguendo lo sviluppo di Laplace per $A$ rispetto la terza colonna otteniamo:

$$det(A)=g\cdot det(A_{33})-z\cdot det(A_{43})$$

Sviluppiamo con Laplace $A_{33},A_{43}$ entrambe rispetto alla terza colonna, ottenendo:

$$det(A)=g\cdot w\cdot det\begin{bmatrix}a&b\\
c&d\end{bmatrix}-z\cdot h\cdot det\begin{bmatrix}a&b\\
c&d\end{bmatrix}=det\begin{bmatrix}a&b\\
c&d\end{bmatrix}\cdot [gw-zh]$$

Ma $[gw-zh]$ è il determinante della matrice:

$$\begin{bmatrix}g&h\\
z&w\end{bmatrix}$$

Allora:

$$det(A)=\begin{bmatrix}a&b\\
c&d\end{bmatrix}\cdot 
\begin{bmatrix}g&h\\
z&w\end{bmatrix}$$

Ragionamento analogo per $det(B)$ sviluppando la terza o quarta riga con Laplace.

**Comment by cornflexxx** [comment]
### Soluzione alternativa 
----------------------------
Riconduco $B$ ad una matrice triangolare superiore $S_B$ , e  calcolo il  $det(S_B)=\prod s_{11}\cdot...\cdot s_{44}$(elt della diagonale):

$$S_B=\begin{vmatrix}
 a&  b&  l&  m\\
 0&  d-cb/a&  n&  p\\
 0&  0&  g&  h\\
 0&  0&  0&  w-zh/g
\end{vmatrix}$$

$$det(B)=1^0\cdot det(S_B)=(ad-cb)\cdot(gw-zh)=det\begin{vmatrix} a& b\\
c& d\end{vmatrix}\cdot det\begin{vmatrix} g& h\\
z& w\end{vmatrix} $$

Faccio la stessa cosa con $A$ in questo caso o scambiando le righe oppure $\to$ matrice triangolare inferiore.

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/11b3dd45-1db7-4716-97cc-71219fdf1377)

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/ed4ee3a9-e61d-4bdf-ae37-83e2ed734fd8)

**Comment by MarcoLinardi** [comment]
- Possiamo vedere la matrice divisa in 4 blocchi, diventando di fatto una matrice 2x2

| A11 | A12 |
| A21 | A22 | 

Quindi det A = det |A11| x det |A22| - det |A12| x det |A21| 
Da qui abbiamo che det A = det |A11| x det |A22|    ,poichè il secondo termine contiene A12 e A21 che contengono tutti zeri, quindi si annulla
Percorso analogo per B
Andando a sostituire le lettere vediamo che la tesi è dimostrata
